proxy/http2/Http2Stream.cc

```diff
@@ -1140,3 +1140,15 @@ Http2Stream::is_first_transaction() const
 {
   return is_first_transaction_flag;
 }
+
+int
+Http2Stream::get_transaction_priority_weight() const
+{
+  return priority_node ? priority_node->weight : 0;
+}
+
+int
+Http2Stream::get_transaction_priority_dependence() const
+{
+  return priority_node && priority_node->parent ? priority_node->parent->id : 0;
```

Comment:
IIUC, `0` is the root node. If the `priority_node` is `nullptr`, this should return `-1` ?

Reply:
Could point.  Will update.